source/Playnite/SortableNameConverter.cs

```diff
@@ -0,0 +1,209 @@
+                }
+                return match.Value;
+            });
+        }
+
+        private static bool MatchComesAfterChapterOrEpisodeOrAtEndOfString(string input, Match match)
+        {
+            bool matchIsAtEndOfString = match.Index + match.Length == input.Length;
+            string theBitImmediatelyPriorToTheMatch = input.Substring(Math.Max(0, match.Index - 9), length: Math.Min(9, match.Index));
+            return matchIsAtEndOfString
+                || theBitImmediatelyPriorToTheMatch.Contains("chapter", StringComparison.InvariantCultureIgnoreCase)
+                || theBitImmediatelyPriorToTheMatch.Contains("episode", StringComparison.InvariantCultureIgnoreCase);
+        }
+
+        private static Dictionary<char, int> RomanNumeralValues = new Dictionary<char, int>
+        {
+            { 'I', 1 }, { 'V', 5 }, { 'X', 10 }, { 'L', 50 }, { 'C', 100 }, { 'D', 500 }, { 'M', 1000 },
+            //unicode uppercase
+            {'Ⅰ', 1}, {'Ⅱ', 2}, {'Ⅲ', 3}, {'Ⅳ', 4}, {'Ⅴ', 5}, {'Ⅵ', 6}, {'Ⅶ', 7}, {'Ⅷ', 8}, {'Ⅸ', 9}, {'Ⅹ', 10}, {'Ⅺ', 11}, {'Ⅻ', 12}, {'Ⅼ', 50}, {'Ⅽ', 100}, {'Ⅾ', 500}, {'Ⅿ', 1000},
+            //unicode lowercase
+            {'ⅰ', 1}, {'ⅱ', 2}, {'ⅲ', 3}, {'ⅳ', 4}, {'ⅴ', 5}, {'ⅵ', 6}, {'ⅶ', 7}, {'ⅷ', 8}, {'ⅸ', 9}, {'ⅹ', 10}, {'ⅺ', 11}, {'ⅻ', 12}, {'ⅼ', 50}, {'ⅽ', 100}, {'ⅾ', 500}, {'ⅿ', 1000},
+            //unicode big/exotic numbers
+            {'ↀ', 1000}, {'ↁ', 5000}, {'ↂ', 10000}, {'Ↄ', 100}, {'ↄ', 100}, {'ↅ', 6}, {'ↆ', 50 }, {'ↇ', 50000}, {'ↈ', 100000 }
+        };
+
+        /// <summary>
+        /// Convert a number from Roman numerals to an integer
+        /// </summary>
+        /// <param name="input">The roman numeral.</param>
+        /// <param name="validate">If false, parse any roman numeral. If true, reject invalid ones.</param>
+        /// <returns>An integer form of the supplied roman numeral, or NULL if the supplied roman numeral is invalid and <paramref name="validate"/> is true.</returns>
+        /// <exception cref="KeyNotFoundException">When the input contains non-numeral characters.</exception>
+        public static int? ConvertRomanNumeralToInt(string input, bool validate = true)
+        {
+            if (string.IsNullOrWhiteSpace(input))
+            {
+                return null;
+            }
+
+            int output = 0;
+            int biggestNumberToTheRight = 0;
+
+            int prevCharGroupLength = 0;
+            int lastNumericValue = 0;
+            for (int i = input.Length - 1; i >= 0; i--)
+            {
+                char c = input[i];
+                int value = RomanNumeralValues[c];
+                bool subtract = value < biggestNumberToTheRight;
+                if (subtract)
+                {
+                    output -= value;
+                }
+                else
+                {
+                    output += value;
+                    biggestNumberToTheRight = value;
+                }
+
+                #region validation
```

Comment:
This region marking doesn't make sense to me, it's almost the entire for loop in it.

Reply:
I found it useful to have the part that does the validation (which yes, is the majority of the loop content) collapsible in order to just look at the calculation part. If you want, this can be done by nesting it in an if(validate) too.